docs/.vuepress/config.js

```diff
@@ -52,17 +52,17 @@ module.exports = {
     custom: true,
     project: {
       name: 'Evmos',
-      denom: 'Evmos',
+      denom: 'EVMOS',
```

Comment:
Should this be all caps?

Reply:
yeah, shouldn't it?

Reply:
I'm confused because denoms are typically all lowercase! EVMOS is the ticker. Name is Evmos. Denom is `evmos` or `Evmos` - if 10**1 then `Evmos` if 10**-18 then `aevmos` if 10**-6 then `uevmos`

`evmos` `osmo` `atom`

Why it's important is because if you look at what it generates, it generates docs that say EVMOS, not Evmos or evmos, and that leads to confusion about how it's pronounced.

Check out what happened on Delphi Digital newsletters:
![image](https://user-images.githubusercontent.com/8366960/150448472-a259786e-4095-42e7-963e-d8ce0798dfd7.png)

Reply:
I think we should also keep the PHOTON denomination for testnet. It's a good one. It's also a way to prevent random chains/scams from showing up with PHOTON.

Reply:
@khoslaventures If the stardard for denom on other chains is all lower case we should stick with `evmos`. 

@fedekunze I also like the idea of photons for the testnet. It's easier to differentiate.